cpp/src_prims/stats/minmax.h

```diff
@@ -57,17 +121,29 @@ __global__ void minmaxKernel(const T* data, const int* rowids,
     }
     int index = row + col * row_stride;
     T coldata = data[index];
-    myAtomicMin(&s_min[col], coldata);
-    myAtomicMax(&s_max[col], coldata);
+    if (use_bits) {
+      if (!isnan(coldata)) {
+        atomicMinBits<T, E>(&s_min[col], coldata);
+        atomicMaxBits<T, E>(&s_max[col], coldata);
+      }
+    } else {
+      myAtomicMin(&s_min[col], coldata);
```

Comment:
Do we need to keep the old path alive in light of this faster bit-twiddling based min/max approach?

Reply:
The only other place that uses the `atomicCAS()`-based min/max atomics is the test (minmax.cu). So I can move those atomics there, and remove the old path from the implementation.

Reply:
That makes sense to do. Thanks.